array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,14 @@
+/* exported compact */
+
+function compact(array) {
+  var newArr = [];
+
+  array.forEach(a => {
```

Comment:
interesting approach! Nothing wrong with it as long as you know how all the parts are working. If you don't I recommend asking about it! I bring that up because you don't see a lot of arrow functions and forEach used at this stage.

Reply:
i was looking around and found a article that was talking about forEach and thought it would work and i tried it and it did and then i went back and broke down everything so i would understand it.